Examples/loadExampleData.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-mongo home --eval "db.kittens.insert([{\"name\":\"roscoe\",\"color\":\"tan\"},{\"name\":\"chester\",\"color\":\"orange\"}])"
+mongo home --eval "db.kittens.insert([{\"name\":\"roscoe\",\"color\":\"orange\"},{\"name\":\"chester\",\"color\":\"tan\"}])"
```

Comment:
wanted the example data to be accurate 😛

Reply:
my apologies to the boys!

Reply:
🐈🐈